src/v/archival/upload_housekeeping_service.cc

```diff
@@ -145,7 +145,9 @@ void upload_housekeeping_service::rearm_idle_timer() {
 
 void upload_housekeeping_service::idle_timer_callback() {
     vlog(_ctxlog.debug, "Cloud storage is idle");
-    if (_workflow.state() == housekeeping_state::idle) {
+    if (
+      _workflow.state() == housekeeping_state::idle
+      || _workflow.state() == housekeeping_state::pause) {
```

Comment:
I'm fuzzy on what the expected behavior is, the "pause" state sounds like it's _meant_ to stay paused until the epoch timer?  I would defer to @Lazin on the difference between idle+epoch timers (maybe you guys already synced)

Reply:
I asked Evgeny out-of-band and he's ok with this. I think the intention was always to be able to transtition from pause to active (at least according to the state machine diagram in the header file).

Reply:
exactly :)